.github/workflows/ci.yml

```diff
@@ -127,8 +127,8 @@ jobs:
           - {php-version: "7.3", db-image: "mariadb:10.5", always: false}
           - {php-version: "7.4", db-image: "mariadb:10.5", always: false}
           - {php-version: "7.4", db-image: "mariadb:10.1", always: false}
-          - {php-version: "7.4", db-image: "mysql:5.6", always: false}
-          - {php-version: "7.4", db-image: "mysql:8.0", always: false}
+          - {php-version: "7.4", db-image: "mysql:5.6", always: true}
+          - {php-version: "7.4", db-image: "mysql:8.0", always: true}
```

Comment:
This should not be included in PR, will be removed once validated.

Reply:
You can proceed ;)

Reply:
Done.